Formulario.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE-edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ejercicio Formulario</title>
+    <link rel="stylesheet" href="instalar%20bp-ac/css/bootstrap.min.css">
+    <script src="instalar%20bp-ac/js/bootstrap.bundle.js"></script>
+    <style>
+        body{
+            background-color: #badce3;
+        }
+    </style>
+
+</head>
+<body>
+<section>
+    <div class="text-black text-center ">
+        <p style="font-size: x-large">EJERCICIO FORMULARIO HTML</p>
+         </div>
+        <div class="container">
```

Comment:
Cuida las identaciones, es importante que para mejor legibilidad del archivo respetes las identaciones (son los espacios de donde empieza el text), los nodos hijos lleva una identacion mas que los padres, esto para representar su jerarquia, si esta mal identado no pasa nada en el navegador igual vas a poder ver la pagian pero para lectura es buena practica, te dejo un ejemplo de lo que te comento

```suggestion
</head>
  <body>
    <section>
      <div class="text-black text-center ">
        <p style="font-size: x-large">EJERCICIO FORMULARIO HTML</p>
     </div>
     <div class="container">
```